src/f3dasm/_src/experimentdata/experimentdata.py

```diff
@@ -171,6 +177,20 @@ def wrapper_func(self, *args, **kwargs) -> None:
 
         return wrapper_func
 
+    #                                                                    Properties
+    # =============================================================================
+
+    @property
+    def index(self) -> pd.Index:
+        """Returns the index of the ExperimentData
+
+        Returns
+        -------
+        pd.Index
+            The indices of the experiments
+        """
```

Comment:
Method and return docstrings don't seem consistent with each other and are not clear.

Reply:
I made the description of this docstring more descriptive and added clarity to the return type